doc/lsp.txt

```diff
@@ -215,7 +215,9 @@ accepts a list of LSP servers with the above information.
 			The popup is also automatically displayed in insert
 			mode after entering a symbol name followed by a
 			separator (e.g. a opening parenthesis). Unless if: >
-			let g:LSP_Show_Signature = v:false
+			let g:LSP_Show_Signature = v:false. If you set
+			g:LSP_Show_Signature = 1 the signature will be echoed
```

Comment:
This should be g:LSP_Echo_Signature = v:true?

Reply:
Yes it's an error. Il'll fix this.